.gitignore

```diff
@@ -51,6 +51,7 @@ build
 #tools
 .vscode
 .idea
+CMakeLists.txt.user
```

Comment:
Is this necessary?

Reply:
yes, I work with qtcreator, this this the pendent to .vscode